src/base.c

```diff
@@ -536,10 +538,21 @@ base_alloc_edata(tsdn_t *tsdn, base_t *base) {
 	if (edata == NULL) {
 		return NULL;
 	}
+	if (config_stats) {
+		base->edata_allocated += sizeof(edata_t);
```

Comment:
The size accounted for should be the "usable size" (same for the rtree case), since the input size is always rounded up to a size class (even though we usually craft the metadata so that size == usize), i.e.
https://github.com/jemalloc/jemalloc/blob/005f20aa7fdef1be6f9fe46e4f2e7b88177a9f21/src/base.c#L487

Let's add another `size_t *ret_usize` arg to `base_alloc_impl` so that the usable size can be returned to the caller.